src/docker-in-docker/install.sh

```diff
@@ -20,6 +20,7 @@ INSTALL_DOCKER_COMPOSE_SWITCH="${INSTALLDOCKERCOMPOSESWITCH:-"true"}"
 MICROSOFT_GPG_KEYS_URI="https://packages.microsoft.com/keys/microsoft.asc"
 DOCKER_MOBY_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal jammy noble"
 DOCKER_LICENSED_ARCHIVE_VERSION_CODENAMES="bookworm buster bullseye bionic focal hirsute impish jammy noble"
+TOGGLE_IP6TABLES_SETTING="${IP6TABLES:-"true"}"
```

Comment:
Can we add a test with `ip6tables:false` and test if the scenario mentioned by user works with these changes?

Reply:
done